sonic_platform_base/chassis_base.py

```diff
@@ -60,6 +60,9 @@ class ChassisBase(device_base.DeviceBase):
     # Object derived from eeprom_tlvinfo.TlvInfoDecoder indicating the eeprom on the chassis
     _eeprom = None
 
+    # System status LED
+    _led = None
```

Comment:
Should we make the variable name more descriptive, like `_status_led`?

Reply:
sure